src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -120,13 +120,13 @@ def collect_tool_methods_in_module(m):
     return tools
 
 
-def _parse_tool_from_function(f):
+def _parse_tool_from_function(f, should_gen_custom_type=False):
```

Comment:
What is this param for? custom connection or custom strong type conn?

Reply:
`gen_custom_type_conn`

Reply:
updated